modules/motions/ui/MotionFormStartNew/Parts/StartSDVTNodeOperatorsAdd.tsx

```diff
@@ -82,7 +86,7 @@ export const formParts = ({
       nodeOperators: [
         { name: '', rewardAddress: '', managerAddress: '' },
       ] as NodeOperator[],
-      nodeOperatorsCount: [{ current: '' }] as { current: string }[],
+      nodeOperatorsCount: [{ count: '' }] as { count: string }[],
```

Comment:
`nodeOperatorsCount` field could be set as just a string, there's no need to declare it as an array imo.